edi-tools/modules/codegen/maincode_gen.bal

```diff
@@ -37,6 +42,11 @@ public isolated function fromEdiString(string ediText, EDI_NAME ediName) returns
     return ediDeserialize(ediText);
 }
 
+# Convert Ballerina record to EDI string.
+# 
+# + data - Ballerina record to be converted
+# + ediName - EDI type name
+# + return - EDI string
```

Comment:
Shouldn't this be string or error? it just says EDI string

Reply:
Changed with 1330f90e144a8514f5a75f7fd1d041a44035e7ba.